contracts/chain-codec/src/state.rs

```diff
@@ -0,0 +1,12 @@
+use axelar_wasm_std::hash::Hash;
+use cosmwasm_schema::cw_serde;
+use cosmwasm_std::Addr;
+use cw_storage_plus::Item;
+
+pub const CONFIG: Item<Config> = Item::new("config");
```

Comment:
could you set this private and instead access it through state functions? we are  trying to use that pattern going forward. See the interchain-token-service contract as an example: https://github.com/axelarnetwork/axelar-amplifier/blob/b6db3a422264fe7d19971a54ef1cbef27838c275/contracts/interchain-token-service/src/state.rs#L169-L177